rdm/pull.py

```diff
@@ -0,0 +1,56 @@
+'''
+Functions that parse the given project management service and produce
+`requirements.yml` and `problem_reports.yml` files.  The selected project
+management backend is specified in the project configuration file.
+'''
+import os
+from getpass import getpass
+
+from github import Github
+import yaml
+
+from rdm.util import get_project_settings
+
+
+def pull_requirements_and_reports(system_yml_path):
+    settings = get_project_settings(system_yml_path)
+    if settings['software_requirements_location'] == 'GitHub':
+        output_dir = os.path.dirname(system_yml_path)
+        pull_from_github(settings, output_dir)
+    else:
+        raise ValueError("Specified software requirements location not yet supported.")
+
+
+def pull_from_github(settings, output_dir):
+    g = authenticate_github()
+    repository = g.get_repo(settings['repository'])
+
+    requirements = {issue.number: {'title': issue.title, 'description': issue.body.replace('\r', '')}
```

Comment:
I'm guessing the `replace('\r', '')` is to convert form windows newlines to linux newlines?  Maybe it would be worth writing a little function for this, just for code readability?